src/prefect/deployments/runner.py

```diff
@@ -428,7 +428,7 @@ def _construct_deployment_schedules(
         else:
             return [create_deployment_schedule_create(schedule)]
 
-    def _set_defaults_from_flow(self, flow: "Flow"):
+    def _set_defaults_from_flow(self, flow: "Flow[Any, Any]"):
```

Comment:
The first parameter is a ParamSpec so the wildcard annotation should actually be `Flow[..., Any]`, much like a Callable. See [the typing documentation](https://typing.readthedocs.io/en/latest/spec/callables.html#meaning-of-in-callable):

> The ... syntax can also be used to provide a specialized value for a ParamSpec in a generic class or type alias.